languages/java/src/main/java/com/bitwarden/sdk/ProjectsClient.java

```diff
@@ -0,0 +1,94 @@
+package com.bitwarden.sdk;
+
+import com.bitwarden.sdk.schema.*;
+
+import java.util.UUID;
+
+public class ProjectsClient {
+
+    private CommandRunner commandRunner;
+
+    ProjectsClient(CommandRunner commandRunner) {
+        this.commandRunner = commandRunner;
+    }
+
+    public ProjectResponse get(UUID id) {
+        Command command = new Command();
+        ProjectsCommand projectsCommand = new ProjectsCommand();
+        ProjectGetRequest projectGetRequest = new ProjectGetRequest();
+        projectGetRequest.setID(id);
+        projectsCommand.setGet(projectGetRequest);
+        command.setProjects(projectsCommand);
+        ResponseForProjectResponse response = commandRunner.runCommand(command,
+            BitwardenClient.throwingFunctionWrapper(Converter::ResponseForProjectResponseFromJsonString));
+        if (response == null || !response.getSuccess()) {
+            throw new BitwardenClientException(response != null ? response.getErrorMessage() : "Project not found");
+        }
+        return response.getData();
+    }
+
+    public ProjectResponse create(UUID organizationId, String name) {
+        Command command = new Command();
+        ProjectsCommand projectsCommand = new ProjectsCommand();
+        ProjectCreateRequest projectCreateRequest = new ProjectCreateRequest();
+        projectCreateRequest.setOrganizationID(organizationId);
+        projectCreateRequest.setName(name);
+        projectsCommand.setCreate(projectCreateRequest);
+        command.setProjects(projectsCommand);
+        ResponseForProjectResponse response = commandRunner.runCommand(command,
+            BitwardenClient.throwingFunctionWrapper(Converter::ResponseForProjectResponseFromJsonString));
+        if (response == null || !response.getSuccess()) {
+            throw new BitwardenClientException(response != null ? response.getErrorMessage() : "Project create failed");
+        }
+        return response.getData();
```

Comment:
Can we improve. the readability of these functions? I think newlines would go a long way towards making these more readable.

```suggestion
        Command command = new Command();
        ProjectsCommand projectsCommand = new ProjectsCommand();
        ProjectCreateRequest projectCreateRequest = new ProjectCreateRequest();
        projectCreateRequest.setOrganizationID(organizationId);
        projectCreateRequest.setName(name);
        projectsCommand.setCreate(projectCreateRequest);
        command.setProjects(projectsCommand);

        ResponseForProjectResponse response = commandRunner.runCommand(command,
            BitwardenClient.throwingFunctionWrapper(Converter::ResponseForProjectResponseFromJsonString));

        if (response == null || !response.getSuccess()) {
            throw new BitwardenClientException(response != null ? response.getErrorMessage() : "Project create failed");
        }

        return response.getData();
```